### ISSUE TITLE
Accessibility - Main Country Screen - Poor Color Contrast of Tabs

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Child-Labor-Android#7]
It's hard to tell that there are three tabs on the Android application because of poor color contrast and no outline like on the iOS application.

**Comment by aquilent-dgemoets** [comment]
This is the default Android tab UI element and will be familiar to Android users.  That said, if time permits, we will investigate alternative tab styling supported out-of-the-box by the platform.

**Comment by jfc3-dol** [comment]
It doesn't matter if these are standard Android elements if the color contrast is poor it makes it difficult for people with visual issues to read the text, which is what you want people to do and it only is worse if a person is viewing information on phone or tablet is bright sunlight. it's difficult to tell which tab is active one.

**Comment by jfc3-dol** [comment]
The light blue text on the dark blue background for the tabs doesn't have enough color contrast.

**Comment by jfc3-dol** [comment]
Plus, there is only a thin white line under the current item that is difficult to see if you have visual issues or are not paying attention.

**Comment by jfc3-dol** [comment]
It doesn't matter if these are standard Android elements if the color contrast is poor it makes it difficult for people with visual issues to tell where one item starts and stops. You want people to use your application. It only is worse if a person is viewing information on phone or tablet is bright sunlight. It's difficult to tell which tab is active.

**Comment by NJICF** [comment]
The current tab selected text and tab selected indicator color is #999999

**Comment by jfc3-dol** [comment]
@NJICF,

Thanks, what is the hex value for the dark blue?

**Comment by NJICF** [comment]
The background dark blue is #003354

**Comment by jfc3-dol** [comment]
Thanks, for the dark blue hex color.

Please use a light gray of **#cccccc** which have a color contrast ratio of 8.6:1. Along with being different enough than the other white tab text. Plus, enough or close for the line under the text not to match the white below. It will be enough to let people read the text of the current tab item between it and the dark blue background.

**Comment by NJICF** [comment]
Light gray of #cccccc color lines have been added.

**Comment by jfc3-dol** [comment]
@NJICF,

If you are done making changes to the Sweat and Toil Android application I will have @giridhar create and place the updates to our devices.

**Comment by NJICF** [comment]
@jfc3-dol,

Yes. The tab colors have been changed,

**Comment by jfc3-dol** [comment]
It's now better and easier to tell which tab of the three is selected. So I'm going to close the issue.
